exporter/local/export.go

```diff
@@ -61,6 +61,9 @@ func (e *localExporterInstance) Name() string {
 }
 
 func (e *localExporterInstance) Export(ctx context.Context, inp exporter.Source) (map[string]string, error) {
+	if inp.Metadata == nil {
+		inp.Metadata = make(map[string][]byte)
+	}
```

Comment:
The local exporter doesn't seem to use `inp.Metadata` anywhere and there is no `e.meta` to merge into it anyway. I'll drop this (and update the commit message).